extensions/wikia/PhalanxII/fallback/hooks/ContentBlock.class.php

```diff
@@ -120,36 +120,36 @@ static public function genericContentCheck( $content ) {
 	 * @param $
 	 * @return
 	 */
-	private static function applyWhitelist($text) {
+	private static function applyWhitelist( $text ) {
 		wfProfileIn( __METHOD__ );
 
-		//TODO: add short memcache here?
-		if (is_null(self::$whitelist)) {
-			$whitelist = wfMsgForContent('Spam-whitelist');
-			if (wfEmptyMsg('Spam-whitelist', $whitelist)) {
+		// TODO: add short memcache here?
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1135)